src/modules/scheduler/services/index.ts

```diff
@@ -0,0 +1,42 @@
+import { WalletService } from "@/modules/api/wallet/services";
+import { PrismaService } from "@/modules/core/prisma/services";
+import { Inject, Injectable, forwardRef } from "@nestjs/common";
+import { Cron } from "@nestjs/schedule";
+
+@Injectable()
+export class SchedulerService {
+    constructor(
+        private prisma: PrismaService,
+        @Inject(forwardRef(() => WalletService))
+        private walletService: WalletService
+    ) {}
+    @Cron("59 23 * * *", {
+        timeZone: "Africa/Lagos",
+    })
+    async aggregateWalletOpeningBalance() {
```

Comment:
Did you test this function to run automatically at set interval?

Reply:
Yes set the time to 5 and 45seconds respectively. the function ran at the specified time.